src/Form/BaseEventCommandType.php

```diff
@@ -18,8 +18,9 @@ class BaseEventCommandType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $beginDate = $builder->getData() ? $builder->getData()->getBeginAt() : null;
         $dateTimeOptions = [
-            'min_date' => new \DateTime('now'),
+            'min_date' => $beginDate ? $beginDate : new \DateTime('now'),
```

Comment:
if I want to edit my event and move back the beginning date, how can I do it?

Reply:
Good question😄